apps/full-site-editing/full-site-editing-plugin/newspack-blocks/index.php

```diff
@@ -64,16 +64,14 @@ function newspack_blocks_block_args( $args, $name ) {
 	wp_register_style( $block_prefix . 'editor', $editor_style, array(), NEWSPACK_BLOCKS__VERSION );
 
 	// View script.
-	if ( 'carousel' !== $name ) {
-		$script_data = require NEWSPACK_BLOCKS__BLOCKS_DIRECTORY . $block_prefix . 'view.asset.php';
-		wp_register_script(
-			$block_prefix . 'view',
-			plugins_url( NEWSPACK_BLOCKS__BLOCKS_DIRECTORY . $block_prefix . 'view.js', __FILE__ ),
-			$script_data['dependencies'],
-			$script_data['version'],
-			true
-		);
-	}
+	$script_data = require NEWSPACK_BLOCKS__BLOCKS_DIRECTORY . $block_prefix . 'view.asset.php';
```

Comment:
This will work, but in general we've been trying to keep the actual feature changes outside of version bump PR. It makes it easier to scan the history and determine which changes ended up in a particular version.

Reply:
@vindl Oh, I assumed fixes that we were finding while testing would be ok as they "belong" to the version bump. What I hear you saying is you'd prefer this specific change to be pulled out into a different PR and merged to master independently first?

Reply:
Handling fixes in https://github.com/Automattic/wp-calypso/pull/41817 and https://github.com/Automattic/wp-calypso/pull/41818